docs(installation): add minimum hardware requirements (#137)
autowarefoundation/autoware-documentation: docs/installation/index.md

- amd64
- arm64

### Minimum hardware requirements

!!! info

    Autoware is scalable and may work with less powerful hardware or distributed hardware under customization. The below minimum hardware requirements are just recommendation in general.

- CPU with 8 cores
- 16GB RAM

> Performance will be improved with more cores, RAM and a higher-spec graphics card or GPU core.

- [Optional] NVIDIA GPU (4GB RAM)

> > Although GPU is not required to run basic functionality, it is mandatory to enable the following nerual network related functions:

Reviewer: typo?

```suggestion
> Although GPU is not required to run basic functionality, it is mandatory to enable the following neural network related functions:
```

Author: Yes, fixed.

>     Autoware is scalable and may work with less powerful hardware or distributed hardware under customization. The below minimum hardware requirements are just recommendation in general.
> 
> - CPU with 8 cores
> - 16GB RAM
> 
> > Performance will be improved with more cores, RAM and a higher-spec graphics card or GPU core.
> 
> - [Optional] NVIDIA GPU (4GB RAM)
> 
> > Although GPU is not required to run basic functionality, it is mandatory to enable the following neural network related functions:
> >
> > - LiDAR based object detection
> > - Camera based object detection
> > - Traffic light detection and classification

Reviewer 2: - Simplified the wording of the first sentence of the note and added a line break.
- Moved the information about performance to the note
- Removed the quote formatting in the text (copy/paste from the issue comment perhaps?)  
- Added a link to the "Running Autoware without CUDA" how-to guide

```suggestion
    Autoware is scalable and can be customized to work with distributed or less powerful hardware. 
    The minimum hardware requirements given below are just a general recommendation.
    However, performance will be improved with more cores, RAM and a higher-spec graphics card or GPU core.

- CPU with 8 cores
- 16GB RAM
- [Optional] NVIDIA GPU (4GB RAM)

Although not required to run basic Autoware functionality, a GPU is mandatory to enable the following neural network related functions:

- LiDAR based object detection
- Camera based object detection
- Traffic light detection and classification

For details of how to enable object detection and traffic light detection/classification without a GPU, refer to the [Running Autoware without CUDA how-to guide](https://autowarefoundation.github.io/autoware-documentation/main/how-to-guides/running-autoware-without-cuda/)
```

- amd64
- arm64

### Minimum hardware requirements

!!! info

    Autoware is scalable and can be customized to work with distributed or less powerful hardware.
    The minimum hardware requirements given below are just a general recommendation.
    However, performance will be improved with more cores, RAM and a higher-spec graphics card or GPU core.

- CPU with 8 cores
- 16GB RAM
- [Optional] NVIDIA GPU (4GB RAM)

> Although GPU is not required to run basic functionality, it is mandatory to enable the following neural network related functions:
>
> - LiDAR based object detection
> - Camera based object detection
> - Traffic light detection and classification

> For details of how to enable object detection and traffic light detection/classification without a GPU, refer to the [Running Autoware without CUDA how-to guide](https://autowarefoundation.github.io/autoware-documentation/main/how-to-guides/running-autoware-without-cuda/)

Reviewer: [nits] 
- I think we can drop the words `how-to guide`.
- Missing period?

```suggestion
For details of how to enable object detection and traffic light detection/classification without a GPU, refer to the [Running Autoware without CUDA](https://autowarefoundation.github.io/autoware-documentation/main/how-to-guides/running-autoware-without-cuda/).
```
